spec/lib/dice_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe Dice do
+  describe "#roll_20_sided_dice" do
```

Comment:
Are we going to use `#` prefixes to indicate class methods? Or is something else more idiomatic RSpec?